server/curfu/routers/utilities.py

```diff
@@ -51,6 +52,32 @@ def get_mane_transcripts(request: Request, term: str) -> Dict:
         return {"transcripts": transcripts}
 
 
+@router.get(
+    "/api/utilities/get_transcripts_for_gene",
+    operation_id="getTranscriptsFromGene",
+    response_model=GetGeneTranscriptsResponse,
+    response_model_exclude_none=True,
```

Comment:
needs a [tag](https://fastapi.tiangolo.com/tutorial/metadata/#use-your-tags) -- defined in schemas.py.